elastic/backport-active/action.yml

```diff
@@ -0,0 +1,36 @@
+name: 'Backport Action'
+description: 'Automatically backport changes to specified branches based on PR labels'
+
+inputs:
+  github_token:
+    description: 'GitHub token for authentication'
+    required: true
+    default: ${{ github.token }}
+  backports_url:
+    description: 'URL to fetch the backport branches configuration JSON'
+    required: true
+  
+runs:
+  using: 'composite'
+  steps:
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.10'
```

Comment:
Can we use python `3.13` instead to have a more active version for bug fixes and security?

See EOL it https://devguide.python.org/versions/ 

<img width="844" alt="image" src="https://github.com/user-attachments/assets/2f0d05a1-4925-4017-a53a-213dfdb0a7e3" />